src/App.css

```diff
@@ -1,38 +1,47 @@
+*,
+::before,
+::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
 .App {
   text-align: center;
+  display: flex;
+  flex-direction: column;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.app__header {
+  position: fixed;
+  padding: 30px;
+  width: 100%;
+  background-color: white;
+  box-shadow: rgba(50, 50, 105, 0.15) 0px 2px 5px 0px,
+    rgba(0, 0, 0, 0.05) 0px 1px 1px 0px;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.app__main {
+  height: 100vh;
+  padding: 160px;
+  overflow-y: scroll;
```

Comment:
Hi Debs, I just stumbled upon the problem, that this works for desktop browsers but not on mobile devices because the viewport height includes the address bar etc. [more here!](https://chanind.github.io/javascript/2019/09/28/avoid-100vh-on-mobile-web.html)
I am looking for a workaround right now to fix this problem.

Reply:
oh, good to know. Thank I will have a look,too.

Reply:
Found this solution and it works for me, tested on iPhone Safari Browser:
https://allthingssmitty.com/2020/05/11/css-fix-for-100vh-in-mobile-webkit/

just add these two lines in your body element in the index.css:
```
  min-height: 100vh;
  min-height: -webkit-fill-available;
```

Reply:
thanks! <3